korge/src/jvmTest/kotlin/com/soywiz/korge/view/camera/CameraTest.kt

```diff
@@ -0,0 +1,87 @@
+package com.soywiz.korge.view.camera
+
+import com.soywiz.klock.*
+import com.soywiz.korge.*
+import com.soywiz.korge.input.*
+import com.soywiz.korge.time.*
+import com.soywiz.korge.tween.*
+import com.soywiz.korge.view.*
+import com.soywiz.korgw.*
+import com.soywiz.korim.color.*
+import com.soywiz.korma.geom.*
+import kotlinx.coroutines.*
+import kotlin.math.*
+
+fun main(): Unit = runBlocking { korge() }
+
+//TODO: Move completed sample to korge-samples
+suspend fun korge() = Korge(quality = GameWindow.Quality.PERFORMANCE, title = "Camera test", bgcolor = Colors.WHITE) {
+
+    onClick { speed = abs(speed - 1.0) }
+
+    fun elements() = Container().apply {
+        solidRect(400.0, 400.0, Colors.YELLOW)
+        solidRect(350.0, 350.0, Colors.YELLOWGREEN)
+        solidRect(300.0, 300.0, Colors.GREENYELLOW)
+        solidRect(250.0, 250.0, Colors.GREEN)
+        polygon(50.0, 7, Colors.DARKGREY).position(200.0, 100.0)
+    }
+
+    val cam = Camera(0.0, 0.0, 400.0, 400.0)
```

Comment:
The cam does not behave as I think it would.
```
val cam = Camera(.0, .0, cont.width, cont.height )
	cont.camera = cam
	cam.follow(player.sprite)
```
produces the display below (with the 2 times scaling issue on mac).
But 
```
val cam = Camera(.0, .0, cont.width*2, cont.height*2 )
	cont.camera = cam
	cam.follow(player.sprite)
```
produces an empty screen (only filled with the KorGE bgcolor)